build/bazel/remote/execution/v2/remote_execution.proto

```diff
@@ -1407,6 +1412,11 @@ message ExecuteOperationMetadata {
   // being executed.
   Digest action_digest = 2;
 
+  // A unique identifier for an ongoing execution of the action, which will
+  // also be returned as `ActionResult.execution_metadata.execution_uuid`.
+  // If the execution is restarted, the server SHOULD provide a new identifier.
+  string execution_uuid = 5;
```

Comment:
I think it's better to stick to the generic "id" rather than the specific "uuid". I don't think we need to constrain implementors specifically to UUIDs.

Reply:
Correct, I'll change that. The description of "unique id" is enough.